cmd/podman/main.go

```diff
@@ -43,9 +43,7 @@ func main() {
 		(len(os.Args[0]) > 0 && filepath.Base(os.Args[0][1:]) == registry.PodmanSh) {
 		shell := strings.TrimPrefix(os.Args[0], "-")
 
-		// The wait timeout will soon be made configurable via the
-		// upcoming `podmansh_timeout` option in containers.conf
-		args := []string{shell, "exec", "-i", "--wait", "30"}
+		args := []string{shell, "exec", "-i", "--wait", fmt.Sprintf("%d", registry.PodmanConfig().ContainersConfDefaultsRO.Engine.PodmanshTimeout)}
```

Comment:
use strconv.FormatUint(...,10) over fmt.Sprintf()

Reply:
done

Reply:
thanks for this suggestion @Luap99